locales/en.default.schema.json

```diff
@@ -2008,7 +2008,7 @@
             },
             "show_gift_card_recipient": {
               "label": "Show recipient information form for gift card products",
-              "info": "Gift card products can optionally be sent direct to a recipient along with a personal message. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/add-gift-card-recipient-fields)"
+              "info": "Gift card products can optionally be sent direct to a recipient on a scheduled date along with a personal message. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/add-gift-card-recipient-fields)"
```

Comment:
Might be a little too late to change for the potential release this week. So maybe we can make that change you suggested @katycobb if @mzarud's team is good with it as well in a follow up for the next release 🤔